api/channel.go

```diff
@@ -1300,3 +1301,27 @@ func viewChannel(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	ReturnStatusOK(w)
 }
+
+func getChannelMembersByIds(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
The only thing that worries me is how often this endpoint is going to be hit, depending on that we might need to add some caching or handle etags, apart from that looks good